backend/prisma/schema.prisma

```diff
@@ -118,23 +118,22 @@ model Contact {
   owner              User                      @relation(fields: [ownerID], references: [id])
   importantDateEvent ImportantDateEvent[]
   relationships      UserContactRelationship[]
-  Plan               Plan?                     @relation(fields: [planId], references: [id])
-  planId             String?
```

Comment:
Here, we should still have `Plan  Plan?  @relation(fields: [planId], references: [id])` and  `planId   String?`

Reply:
We got this fixed so ima just ignore this